README.md

```diff
@@ -97,7 +102,34 @@ The SDO owner services are packaged as a single docker container that can be run
    ```bash
    curl -sS -w "%{http_code}" -X POST $SDO_RV_URL/mp/113/msg/20 | jq
    ```
+   
+### <a name="gen-keypair"></a>Generate Owner Key Pairs
+
+If you want to expedite the process of creating key pairs, you can run `keys/generate-key-pair.sh`
```

Comment:
Again, not an option, but a directive. I suggest: For production use of SDO, you need to create 3 key pairs and import them into the owner services container. These key pairs enable you to securely take over ownership of SDO ownership vouchers from SDO-enabled device manufacturers, and to securely configure your booting SDO devices. Use the provided `generate-key-pair.sh` script to easily create the necessary key pairs. (If you are only trying out SDO in a dev/test environment, you can use the sample default key pairs and skip this section. You can always add your own key pairs later.)